admin-site/src/Occasion.svelte

```diff
@@ -357,9 +355,7 @@ function deleteOccasion() {
   modalTitle="Close Occasion">
   
   <div slot="modalBody">
-    <!-- {#if gotEvents} -->
       Are you sure you want to close {focusedEvent.label} - {formattedStartTime}?
```

Comment:
For messages that include a variable, you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals to still pull them into a separate file along with simpler UI messages.

Reply:
Is there a standard format for a file containing UI messages? Maybe at our 1 on 1 we can map it out?

Reply:
Sounds good :)